tests/test_nicenumber.py

```diff
@@ -50,3 +50,23 @@ def test_to_human():
         else:
             assert result == expected_result
 
+
+
+def test_to_color():
+    """ Test to_color() function"""
+
```

Comment:
Add test for an expected error with wrong colors.
- could use `f = nn.to_colour` to assign func definition to variable for easier passing to tests

```suggestion

f = nn.to_colour

raises(ValueError, f, colour=['limegreen', 'pink']).match('not available')

```